drivers/hmis/app/graphql/mutations/bulk_review_external_submissions.rb

```diff
@@ -0,0 +1,55 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class BulkReviewExternalSubmissions < CleanBaseMutation
+    argument :external_submission_ids, [ID], required: true
+    field :success, Boolean, null: true
+
+    def resolve(external_submission_ids:)
+      submissions = HmisExternalApis::ExternalForms::FormSubmission.where(id: external_submission_ids)
+      definitions = Hmis::Form::Definition.where(id: submissions.pluck(:definition_id))
+
+      # Submissions could come from different definitions (versions), but should be all the same form identifier
+      identifiers = definitions.pluck(:identifier).uniq
+      error_out('Cannot bulk process submissions from multiple form identifiers: ' + identifiers.inspect) unless identifiers.one?
+
+      project = Hmis::Form::Instance.where(definition: definitions).uniq.sole.entity # Expect only 1 instance/project
```

Comment:
`project = submissions.first.parent_project # can check first because we know they all have the same form identifier`
maybe? I ran into an issue because I had other deactivated rules. This approach ensures that the logic matches